app-text/sigil/sigil-0.9.12.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit cmake-utils python-single-r1
+
+my_pn="Sigil"
+
+DESCRIPTION="Sigil is a multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/"
+SRC_URI="https://github.com/Sigil-Ebook/${my_pn}/archive/${PV}.zip -> ${P}.zip"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	app-text/hunspell
+	dev-libs/boost[threads]
+	dev-libs/libpcre[pcre16]
+	dev-libs/xerces-c[icu]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/cssselect[${PYTHON_USEDEP}]
+	dev-python/cssutils[${PYTHON_USEDEP}]
+	dev-python/html5lib[${PYTHON_USEDEP}]
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]
+	dev-python/regex[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	>=dev-qt/qtconcurrent-5.4:5
+	>=dev-qt/qtcore-5.4:5
+	>=dev-qt/qtgui-5.4:5
+	>=dev-qt/qtprintsupport-5.4:5
+	>=dev-qt/qtwebkit-5.4:5
+	>=dev-qt/qtwidgets-5.4:5
+	>=dev-qt/qtxmlpatterns-5.4:5
+	sys-libs/zlib[minizip]
+"
+DEPEND="${RDEPEND}
+	>=sys-devel/gcc-4.8
```

Comment:
This is not the correct way to force `gcc >=4.8`, please use toolchain-funcs eclass to do that. 

Something like this should work, 
```
pkg_pretend() {
	if tc-is-gcc && [[ $(gcc-version) -lt 4.8 ]] ; then
		die "Active GCC needs to be newer than 4.8"
	fi
}
```
but please test it.